views/note-id.ejs

```diff
@@ -0,0 +1,52 @@
+<%- include('partials/header',{title: `note ${ singleNote.id}`}) %>
+<div class="container">
+  <div class="row">
+    <div class="card d-flex justify-content-center" style="width: 18rem">
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item"><%=singleNote.date%></li>
+        <li class="list-group-item"><%=singleNote.time%></li>
+        <li class="list-group-item"><%=singleNote.day%></li>
+        <li class="list-group-item"><%=singleNote.flock_size%></li>
+      </ul>
+    <div class="card-footer"><%=singleNote.habitat%></div>
+  </div>
+</div>
+
+
+<div class="container mt-4">
+  <div class="row">
+      <form action="/note/<%=singleNote.id  %>/comment" method="POST">
+        <label for="comment">Leave a comment:
+          <textarea type=text id="comment" name="comment" style="width: 100%;"></textarea>
+        </label>
+        <div>
+          <button class= "btn btn-primary" type="submit">Comment</button>
+        </div>
+      </form>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <% for(let i =0; i<comments.length; i+=1){ %>
+        <div class="card mt-1" style="width: 18rem;">
+          <div class="card-body">
+            <span>Posted by: </span>
+            <% if((comments[i].comment_text !== undefined) || (comments[i].comment_text !== null) || (comments[i].comment_text !== ' ')) { %> 
```

Comment:
Is there a way to handle the non-existent comments inside the `index.js` file instead of the EJS so that there isn't the handling logic inside the rendered page?